### ISSUE TITLE
Question: How change a character name?

### ISSUE THREAD

**Issue by koliseo** [olivierkes/manuskript#600]
I want to change the name of one of my characters that appears in a lot of scenes and chapters. Is there any way to do it all at once?

**Comment by mysticpoisen** [comment]
I don't believe so but that is certainly a worthy feature request. A find all and replace feature is essential for any text editor.

**Comment by gedakc** [comment]
Are you seeking one of these already open issues?

- [Issue 145 - global and local search and replace](https://github.com/olivierkes/manuskript/issues/145)
- [Issue 248 - [Feature suggestion] Use symbol to add a reference for a character, place, object in the project](https://github.com/olivierkes/manuskript/issues/248)

If not then please describe your request in more detail.

Note that there are also several other similar requests if you search through the open issues.

**Comment by koliseo** [comment]
I'm afraid that none of theese links is the answer I've searching. I just want a easy and fast way to change names (o whatever word) along the whole book. I have 150 escenes thru 30 chapters. All I get now is a global search that tell me where the word appears... but I still have to navigate from one to another changing the word "manually". What I want is a simple "Replace all appearences of that word in all the book at once" feature.

**Comment by TheJackiMonster** [comment]
I guess this is a request for a global 'search & replace' or a duplicate for issue #341 and this will probably take some time.
